test/extensions/filters/http/custom_response/custom_response_integration_test.cc

```diff
@@ -643,8 +643,7 @@ TEST_P(CustomResponseIntegrationTest, ModifyRequestHeaders) {
   EXPECT_EQ("Modify action response body", response->body());
 }
 
-// Test for HTTP1/2 only.
-// TODO(asingh-g): Fix test suite for HTTP/3.
+// TODO(#26236): Fix test suite for HTTP/3.
 INSTANTIATE_TEST_SUITE_P(
     Protocols, CustomResponseIntegrationTest,
     testing::ValuesIn(HttpProtocolIntegrationTest::getProtocolTestParams(
```

Comment:
nit: should we use getProtocolTestParamsWithoutHTTP3() here too?

Reply:
Whoops, missed one. Fixed now!